packages/pigeon/lib/objc_generator.dart

```diff
@@ -795,6 +800,55 @@ static id GetNullableObjectAtIndex(NSArray *array, NSInteger key) {
       indent.writeln('return sSharedObject;');
     });
   }
+
+  void _writeMethod(ObjcOptions languageOptions, Root root, Indent indent,
```

Comment:
Is this only moved, no other changes? It definitely belongs here, just want to make sure I didn't miss anything.

Reply:
Yes, moved without changes. But this made me remember that I moved this because I was making sure it was safe to change `_callbackForType` without affecting HostApi, and after determining that it was because this could move, I forgot to also move `_callbackForType`, so that'll be moved in the next version (with the type change).

Reply:
Doh, the class division isn't HostApi/FlutterApi. Ignore the part about moving `_callbackForType`.